event/monitoring.go

```diff
@@ -22,7 +22,7 @@ type CommandStartedEvent struct {
 	CommandName  string
 	RequestID    int64
 	ConnectionID string
-	ServerID     *primitive.ObjectID // Only set for servers that are deployed behind a load balancer.
+	ServiceID    *primitive.ObjectID // Only set for servers that are deployed behind a load balancer.
```

Comment:
Should any of these comments referencing "servers" now be "services" instead?

Reply:
The wording is a little tricky here. I don't want to just change everything to "services" but the existing wording isn't great either. I updated the docs to more accurately describe what this field represents.